### PR TITLE
reflect ON DELETE/UPDATE triggers in schema

### PR DESCRIPTION
**Pull request by wmshort**
Addresses remaining enhancement of #193 , namely reflecting ON DELETE and ON UPDATE triggers on foreign key columns in the generated schema

### CODE REVIEW

File: piccolo/apps/schema/commands/generate.py [hunk 1]
```diff
@@ -131,6 +131,41 @@ def get_foreign_key_constraint_name(self, column_name) -> ConstraintTable:
...
+@dataclasses.dataclass
+class TableTriggers:
+    """
+    All of the triggers for a certain table in the database.
+    """
+
+    tablename: str
+    triggers: t.List[Trigger]
+
+    def get_column_triggers(self, column_name) -> t.List[Trigger]:
```
Comment (dantownsend) [inline piccolo/apps/schema/commands/generate.py:154]: Nitpick - type annotation for `column_name`.

File: piccolo/apps/schema/commands/generate.py [hunk 2]
```diff
@@ -131,6 +131,41 @@ def get_foreign_key_constraint_name(self, column_name) -> ConstraintTable:
...
+        triggers = []
+        for i in self.triggers:
+            if i.column_name == column_name:
+                triggers.append(i)
+        return triggers
+    
+    def get_column_ref_trigger(self, column_name, references_table) -> Trigger:
```
Comment (dantownsend) [inline piccolo/apps/schema/commands/generate.py:161]: Nitpick - type annotation for `column_name` and `references_table`.

**Comment by dantownsend** [comment]
It looks great 👍

File: piccolo/apps/schema/commands/generate.py (hunk 1 above)
Comment (wmshort) [reply piccolo/apps/schema/commands/generate.py]: roger that

File: piccolo/apps/schema/commands/generate.py (hunk 2 above)
Comment (wmshort) [reply piccolo/apps/schema/commands/generate.py]: roger that

**Pull request merged**